src/components/MenuItem/MenuItem.jsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { hotPrice } from "../../data"
+
+export class MenuItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bread: "Standard",
+      isHot: false,
+      total: this.props.price
+    };
+
+    this.action = this.action.bind(this);
+    this.changeBread = this.changeBread.bind(this);
+    this.changeHot = this.changeHot.bind(this);
+  }
+
+  changeBread(event) {
+    const { breads, price } = this.props;
+    const bread = breads[event.target.value].name;
+    const breadPrice = breads[event.target.value].price;
+    const total = !this.state.isHot ? price + breadPrice : (price + breadPrice) + hotPrice;
+
+    this.setState({
+      bread,
+      total
+    });
+  }
+
+  changeHot(event) {
+    const isHot = event.target.checked;
+    const total = isHot ? this.state.total + hotPrice : this.state.total - hotPrice;
+
+    this.setState({
+      isHot,
+      total
+    });
+  }
+
+  action() {
+    this.props.addToOrder(
```

Comment:
Destructure props here again man `const { addToOrder, name, total, bread, isHot } = this.props`